test/index.js

```diff
@@ -296,6 +356,50 @@ const testImpl = {
     };
   }),
 
+  helloCommonContext: sinon.spy(function({ name }) {
+    return {
```

Comment:
I think you can just return an empty object here - all the casting etc is being tested elsewhere so you're not adding anything